package.json

```diff
@@ -13,6 +13,7 @@
     "lint:fix": "eslint --ext js,ts,tsx --fix src",
     "test": "jest --env=jsdom",
     "test:ci": "jest --env=jsdom --collectCoverage=true --reporters=default --reporters=jest-junit",
+    "ts:check": "tsc --noEmit",
```

Comment:
@amcdnl should I remove this line?

Reply:
Probably? I'm not sure I follow what its for specifically though....

Reply:
This runs typescript type checking on the whole project and prints errors if any

The modification in `tsconfig.json` fixes typescript module resolve mechanism. As I see you do not use relative imports (e.g. `'import 'PieChart'` instead of `import './src/PieChart'`). So it is better to remove `baseUrl` so that VS Code won't resolve all modules to the relative imports

Reply:
Got it. Ya - I knew it did that checking but wasn't sure why you added it outside of the normal processes.

Reply:
This is for local use or for linting purposes. You do not need to run project build. It only checks types without outputting build